app/src/main/java/io/pslab/fragment/ChannelParametersFragment.java

```diff
@@ -10,6 +10,7 @@
 import androidx.core.content.ContextCompat;
 import androidx.fragment.app.Fragment;
 
+import android.util.Log;
```

Comment:
Looks like this is unused?

Reply:
Oh yes! My bad, I had added these for debugging. I'm removing them